docs/_docs/restapi.adoc

```diff
@@ -506,12 +506,6 @@ http://host:port/ignite?cmd=incr&cacheName={cacheName}&key={incrKey}&init={initi
 |Example
 
 
-|`cacheName`
```

Comment:
Need to remove the "&cacheName={cacheName}" parameter from the URL example as well